pkg/serverless/appsec/appsec.go

```diff
@@ -105,21 +110,27 @@ func (a *AppSec) Close() error {
 
 // Monitor runs the security event rules and return the events as a slice
 // The monitored addresses are all persistent addresses
-func (a *AppSec) Monitor(addresses map[string]any) []any {
+func (a *AppSec) Monitor(addresses map[string]any) (res waf.Result) {
```

Comment:
Nit, but the good C programming principles still apply to Go and are worth it: returning a pointer is more efficient.

Reply:
e3d23db9b084d4894e